src/components/cc-global-header.vue

```diff
@@ -7,8 +7,8 @@
       </a>
     </div>
     <nav class="navbar" :aria-label="ariaPrimaryLabel">
-      <div class="navbar-brand column is-one-fifth">
-        <a class="main-logo" href="/" target="_blank">
+      <div class="navbar-brand is-justify-content-space-between">
+        <a class="main-logo navbar-item" href="/" target="_blank">
```

Comment:
Ideally, we'd be able to resolve this without the `is-justify-content-space-between`. However, I understand we don't want perfect to be the enemy of good. I like the addition of the `navbar-item` class since it follows the Bulma [navbar documentation](https://bulma.io/documentation/components/navbar/).
```suggestion
      <div class="navbar-brand">
        <a class="main-logo navbar-item" href="/" target="_blank">
```

Reply:
If needed, we can merge this pull request as-is. I'd like to dig deeper to understand why we need to revert the navbar flexbox spacing. E.g., is there something in the Vocabulary styles that is breaking the layout?

Reply:
The `is-justify-content-space-between` should not be necessary with the following change to `heading.scss` in the `vocabulary-styles` project:

```css
@import "bulma/sass/components/navbar.sass";

.navbar-brand .navbar-burger {
	margin-left: auto;
}
```

I bel.ieve we should still include `navbar-item` class on the logo.